tests/Application/.env

```diff
@@ -3,7 +3,7 @@
 # https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
 
 ###> symfony/framework-bundle ###
-APP_ENV=test
+APP_ENV=dev
```

Comment:
Is this used in build? If yes, shouldn't it be `test`?

Reply:
Changed back to test